scripts/mkcloud

```diff
@@ -638,6 +620,38 @@ function macfunc()
     printf "52:54:77:77:77:%02x" $1
 }
 
+function get_admin_node_dist()
+{
+    # echo the name of the current dist for the admin node
+    local dist=
+    if iscloudver 7plus; then
+        dist=SLE12SP2
+    elif iscloudver 6plus ; then
+        dist=SLE12SP1
+    elif iscloudver 5plus && [[ $want_sles12 ]] ; then
+        dist=SLE12
+    else
+        dist=SLE11
+    fi
+    echo "$dist"
+}
+
+function get_admin_node_image_name()
+{
+    # get the name of the image to deploy the admin node
+    local dist=`get_admin_node_dist`
+    case $dist in
+        SLE12SP1) image=SLES12-SP1.qcow2 ;;
+        SLE12SP2) image=SLES12-SP2.qcow2 ;;
```

Comment:
I'd reverse the order of these two lines to be consistent with the general ordering that we have here.